### ISSUE TITLE
Lib and egui version

### ISSUE THREAD

**Issue by nrot** [zakarumych/egui-snarl#6]
It`s many changes on last few commits - it time to update version. And time to update egui version to 0.25

**Comment by zakarumych** [comment]
Agree. I want to bump egui version, fix any issues that may appear and publish v0.2
